frontend/src/components/MeetingRoom/Toolbar/Toolbar.tsx

```diff
@@ -70,51 +70,104 @@ const Toolbar = ({
   }, [disconnect]);
   const [openEmojiGridDesktop, setOpenEmojiGridDesktop] = useState<boolean>(false);
 
+  // An array of buttons available for the toolbar. As the toolbar resizes, buttons may be hidden and moved to the
+  // ToolbarOverflowMenu to ensure a responsive layout without compromising usability.
+  const toolbarButtons: Array<ReactElement | false> = [
```

Comment:
This is a little strange to read on GH, but the gist is we're moving all of the buttons we wish to display to this array. Using the `useToolbarButtons` hook, we render whichever buttons we have space for.